Pod/Classes/API/ABMSoundCloudPort.m

```diff
@@ -0,0 +1,287 @@
+//
+//  SoundCloudPort.m
+//  ABMSoundCloudAPI
+//
+//  Created by AndresBrun on 01/10/14.
+//  Copyright (c) 2014 Brun's Software. All rights reserved.
+//
+
+#import "ABMSoundCloudPort.h"
+#import "ABMAuth2Manager.h"
+#import "ABMAuthenticationCredentials.h"
+
+//#import <AFOAuth2Manager/AFOAuth2Manager.h>
+//#import <AFNetworking/AFNetworking.h>
+
+#import "NSError+APISoundCloud.h"
+#import "NSUserDefaults+soundCloudToken.h"
+
+#import "SoundCloudLoginWebViewController.h"
+
+
+
+
+
+NSString *SC_API_URL = @"https://api.soundcloud.com";
+NSString *PROVIDER_IDENTIFIER = @"SoundClount_Crendentials";
+
+
+
+
+
+@interface ABMSoundCloudPort ()
+
+@property (strong, nonatomic) ABMAuth2Manager* oAuth2Manager;
+@property (strong, nonatomic) NSURLSessionDataTask* lastURLSessionDataTask;
+@property (nonatomic, readonly) ABMAuthenticationCredentials *credentials;
+
+@property (weak, nonatomic) UIViewController *supportingVC;
+@property (strong, nonatomic) NSString *redirectURL;
+
+@end
+
+
+
+
+
+@implementation ABMSoundCloudPort
+
+//+(void)initialize
+//{
+//	[NSUserDefaults removeSoundCloudCode];
+//	[ABMAuthenticationCredentials deleteCredentialWithIdentifier:PROVIDER_IDENTIFIER];
+//}
+
+- (instancetype)initWithClientId:(NSString *)clientID
+                    clientSecret:(NSString *)clientSecret {
+	
+	if (self = [super init])
```

Comment:
Try to keep the code style chosen in the project, it is not like the current one is best but it is ugly to have different styles. In this case is because of
```
if() {

}
```

and 

```
self.oAuth2Manager = [ABMAuth2Manager new];
```